trulens_eval/examples/expositional/models/arctic_quickstart.ipynb

```diff
@@ -4,14 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    with over 45,000 students across three campuses in Seattle, Tacoma, and Bothell.
</pre><p>I have a change in the context filtering PR to add more data here which makes the example here more interesting, since now there is only one chunk to retrieve.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Mostly it was mentioned in their doc https://docs.snowflake.com/user-guide/snowflake-cortex/llm-functions#using-snowflake-cortex-llm-functions-with-python (i.e. missing support for chat style input)